app/frontend/src/features/profile/view/About.tsx

```diff
@@ -19,12 +24,28 @@ export default function About({ user }: AboutProps) {
   return (
     <>
       <Typography variant="h1">{OVERVIEW}</Typography>
-      <UserAgeGenderPronouns user={user} />
-      <UserLanguages user={user} />
-      <UserHomeTown user={user} />
-      <UserOccupation user={user} />
-      <UserEducation user={user} />
-      <UserSinceJoined user={user} />
+
+      <LabelAndText
+        label={AGE_GENDER}
+        text={`${user.age} / ${user.gender} ${
+          user.pronouns ? `(${user.pronouns})` : ""
+        }`}
+      />
+      <LabelAndText
+        label={LANGUAGES_FLUENT}
+        text={user.languagesList.toString().replace(",", ", ") || "Not given"}
```

Comment:
Can you extract the "Not given" text to the constant file as well please?

Reply:
Extracted/reused all